dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/DataQueryParams.java

```diff
@@ -690,6 +691,8 @@ protected QueryKey getQueryKey()
             .add( "order", order )
             .add( "timeField", timeField )
             .add( "orgUnitField", orgUnitField )
+            .add( "expressiondimensionitems", this.getExpressionDimensionItems().stream()
```

Comment:
That's good. Could this deserve its own method, like `getExpressionDimensionItemExpressionString` to make the code more readable?

Reply:
Done